test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -1311,13 +1311,15 @@ TEST_F(PickFirstTest, PendingUpdateAndSelectedSubchannelFails) {
       BuildChannel("", response_generator);  // pick_first is the default.
   auto stub = BuildStub(channel);
   // Create a number of servers, but only start 1 of them.
-  CreateServers(10);
+  CreateServers(2);
   StartServer(0);
   // Initially resolve to first server and make sure it connects.
   gpr_log(GPR_INFO, "Phase 1: Connect to first server.");
   response_generator.SetNextResolution({servers_[0]->port_});
   CheckRpcSendOk(DEBUG_LOCATION, stub, true /* wait_for_ready */);
   EXPECT_EQ(channel->GetState(false), GRPC_CHANNEL_READY);
+  ConnectionAttemptInjector injector;
+  auto hold = injector.AddHold(servers_[1]->port_);
   // Send a resolution update with the remaining servers, none of which are
   // running yet, so the update will stay pending.  Note that it's important
```

Comment:
The last sentence of this comment can be removed now.

Reply:
Oops, guess that change didn't make it into the PR. Done.